engine/src/main/resources/templates/Inventory/dailyEmailBody.html

```diff
@@ -28,14 +28,14 @@
             <tr>
             <td class="rh-content__block">
                 <table width:="" 100%;="" class="rh-data-table rh-m-bordered">
-                <thead>              
+                <thead>
                     <tr>
                         <th>Host Name</th>
                         <th>Error</th>
                     </tr>
                 </thead>
                 <tbody>
-                    {#each action.context.inventory.errors.take(10)}
+                    {#each action.context.inventory.errors.take(action.context.inventory.errors.size() > 10 ? 10 : action.context.inventory.errors.size())}
```

Comment:
Though we could use stuff like `java.lang.Math.min` or `Math.min` but looks like it does not come out of the box.